sechub-pds-solutions/owasp-zap/helm/pds-owasp-zap/templates/deployment.yaml

```
@@ -0,0 +1,86 @@
+                      value: "{{ .Values.users.technical.id }}"
+                    - name: TECHUSER_APITOKEN
+                      value: "{{ .Values.users.technical.apiToken }}"
+                    - name: ZAP_API_KEY
+                      value: "{{ .Values.zap.apiKey }}"
+                    - name: PDS_CONFIG_EXECUTE_QUEUE_MAX
+                      value: "{{ .Values.pds.config.execute.queueMax }}"
+                    - name: PDS_CONFIG_EXECUTE_WORKER_THREAD_COUNT
+                      value: "{{ .Values.pds.config.execute.workerThreatCount }}"
+                    - name: ZAP_PROXY_HOST
+                      value: "{{ .Values.zap.proxy.host }}"
+                    - name: ZAP_PROXY_PORT
+                      value: "{{ .Values.zap.proxy.port }}"
+                    {{- if .Values.pds.javaDebug.enabled }}
+                    - name: JAVA_ENABLE_DEBUG
+                      value: "true"
+                    {{- end }}
+                    {{- if .Values.pds.keepContainerAliveAfterPDSCrashed }}
+                    - name: KEEP_CONTAINER_ALIVE_AFTER_PDS_CRASHED
+                      value: "true"
+                    {{- end }}
+                    {{- if .Values.database.postgres.enabled }}
+                    - name: POSTGRES_ENABLED
+                      value: "true"
+                    - name: DATABASE_CONNECTION
+                      value: "{{ .Values.database.postgres.connection }}"
+                    - name: DATABASE_USERNAME
+                      value: "{{ .Values.database.postgres.username }}"
+                    - name: DATABASE_PASSWORD
+                      value: "{{ .Values.database.postgres.password }}"
+                    {{- end }}
+
+                    # Storage priority in order: local, s3, shared volume
+                    # Meaning if local is enabled local will be used,
+                    # regardless of other storage configurations.
+                    {{- if .Values.storage.local.enabled }}
+                    - name: SHARED_VOLUME_UPLOAD_DIR
+                      value: "/shared_volumes/uploads"
+                    {{- else if .Values.storage.s3.enabled }}
+                    - name: S3_ENABLED
+                      value: "true"
+                    - name:  S3_ENDPOINT
+                      value: {{ .Values.storage.s3.endpoint }}
+                    - name: S3_BUCKETNAME
+                      value: {{ .Values.storage.s3.bucketname }}
+                    - name: S3_ACCESSKEY
+                      value: {{ .Values.storage.s3.accesskey }}
+                    - name: S3_SECRETKEY
+                      value: {{ .Values.storage.s3.secretkey }}
+                    {{- else if .Values.storage.sharedVolume.enabled }}
+                    - name: SHARED_VOLUME_UPLOAD_DIR
+                      value: "{{ .Values.storage.sharedVolume.upload.dir }}"
+                    {{- end}}
+                  ports:
+                    - containerPort: 8444
```

Please add startup+liveness probes:
```suggestion
                    - name: pds-https-port
                      containerPort: 8444
                  startupProbe:
                    httpGet:
                      scheme: HTTPS
                      path: /api/anonymous/check/alive
                      port: pds-https-port
                    periodSeconds: 1
                    failureThreshold: 300
                    # probe every 1s x 300 = 5 mins before restart of container
                    successThreshold: 1
                    timeoutSeconds: 1
                  livenessProbe:
                    httpGet:
                      scheme: HTTPS
                      path: /api/anonymous/check/alive
                      port: pds-https-port
                    periodSeconds: 5
                    failureThreshold: 3
                    successThreshold: 1
                    timeoutSeconds: 3
```